package-lock.json

```diff
@@ -38170,6 +38242,25 @@
         }
       }
     },
+    "eslint-plugin-licenses": {
+      "version": "1.0.1",
+      "resolved": "https://registry.npmjs.org/eslint-plugin-licenses/-/eslint-plugin-licenses-1.0.1.tgz",
+      "integrity": "sha512-8ly0AVIri6b7FI1POTztsJxSvssoXLEbyjSRD8wZPkdNKIyNXlcXICGysW8wneU0K8mYxcithckXNpZ/ukvN1w==",
+      "dev": true,
+      "requires": {
+        "@types/eslint": "^8.4.9",
+        "requireindex": "^1.2.0",
+        "typescript": "^4.8.4"
+      },
+      "dependencies": {
+        "typescript": {
```

Comment:
This worries me a bit. Is it better as a `peerDependency` in https://github.com/fallaciousreasoning/eslint-plugin-licenses/blob/main/package.json?

Reply:
It can probably just be a dev dependency, tbh